diff -u -r -U3 libgeotiff-1.7.1/CMakeLists.txt libgeotiff-1.7.1.no-zlib-cmake-package/CMakeLists.txt
--- libgeotiff-1.7.1/CMakeLists.txt	2022-03-10 03:32:14.000000000 -0500
+++ libgeotiff-1.7.1.no-zlib-cmake-package/CMakeLists.txt	2025-12-11 11:55:48.275361577 -0500
@@ -130,10 +130,10 @@
 SET(WITH_ZLIB FALSE CACHE BOOL "Choose if zlib support should be built")
 
 IF(WITH_ZLIB)
-    FIND_PACKAGE(ZLIB NO_MODULE QUIET)
-    if (NOT ZLIB_FOUND)
+    # FIND_PACKAGE(ZLIB NO_MODULE QUIET)
+    # if (NOT ZLIB_FOUND)
       FIND_PACKAGE(ZLIB)
-    endif ()
+    # endif ()
 
     IF(ZLIB_FOUND)
         SET(HAVE_ZIP 1)
