diff --git a/src/MEDLoader/MEDFileMeshLL.hxx b/src/MEDLoader/MEDFileMeshLL.hxx
index 25150e0..5f87350 100644
--- a/src/MEDLoader/MEDFileMeshLL.hxx
+++ b/src/MEDLoader/MEDFileMeshLL.hxx
@@ -21,6 +21,8 @@
 #ifndef __MEDFILEMESHLL_HXX__
 #define __MEDFILEMESHLL_HXX__
 
+#include "MEDLoaderDefines.hxx"
+
 #include "MEDFileBasis.hxx"
 #include "MEDFileMeshElt.hxx"
 
@@ -102,7 +104,7 @@ class StructMeshCls : public MeshOrStructMeshCls
     double checkMeshTimeStep(med_idt fid, const std::string &mName, int nstep, int dt, int it) const;
 };
 
-class MEDFileMeshL2 : public RefCountObject
+class MEDLOADER_EXPORT MEDFileMeshL2 : public RefCountObject
 {
    public:
     MEDFileMeshL2();
diff --git a/src/MEDLoader/MEDFileUtilities.hxx b/src/MEDLoader/MEDFileUtilities.hxx
index 4f38c3c..247d971 100644
--- a/src/MEDLoader/MEDFileUtilities.hxx
+++ b/src/MEDLoader/MEDFileUtilities.hxx
@@ -31,16 +31,16 @@
 
 namespace MEDFileUtilities
 {
-med_access_mode
+MEDLOADER_EXPORT med_access_mode
 TraduceWriteMode(int medloaderwritemode);
-const char *
+MEDLOADER_EXPORT const char *
 GetReadableMEDFieldType(med_field_type ft);
-void
+MEDLOADER_EXPORT void
 CheckMEDCode(int code, med_idt fid, const std::string &msg);
-void
+MEDLOADER_EXPORT void
 CheckFileForRead(const std::string &fileName);
 
-class AutoFid
+class MEDLOADER_EXPORT AutoFid
 {
    public:
     AutoFid(med_idt fid);
diff --git a/src/ParaMEDLoader/CMakeLists.txt b/src/ParaMEDLoader/CMakeLists.txt
index 4db74e1..c7b782d 100644
--- a/src/ParaMEDLoader/CMakeLists.txt
+++ b/src/ParaMEDLoader/CMakeLists.txt
@@ -42,7 +42,7 @@ include_directories(
 
 set(paramedloader_SOURCES ParaMEDLoader.cxx ParaMEDFileMesh.cxx)
 
-add_library(paramedloader ${paramedloader_SOURCES})
+add_library(paramedloader STATIC ${paramedloader_SOURCES})
 target_link_libraries(paramedloader paramedmem medloader ${PLATFORM_LIBS})
 install(
   TARGETS paramedloader
diff --git a/src/ParaMEDMEM/CMakeLists.txt b/src/ParaMEDMEM/CMakeLists.txt
index 449f662..3304274 100644
--- a/src/ParaMEDMEM/CMakeLists.txt
+++ b/src/ParaMEDMEM/CMakeLists.txt
@@ -71,7 +71,7 @@ set(paramedmem_SOURCES
     MPIAccess/MPIAccessDEC.cxx
     CFEMDEC.cxx)
 
-add_library(paramedmem ${paramedmem_SOURCES})
+add_library(paramedmem STATIC ${paramedmem_SOURCES})
 target_link_libraries(paramedmem medcouplingcpp medicoco ${MPI_LIBRARIES})
 install(
   TARGETS paramedmem
