diff --git a/src/ICoCo/CMakeLists.txt b/src/ICoCo/CMakeLists.txt
index 3918a99..a196e93 100644
--- a/src/ICoCo/CMakeLists.txt
+++ b/src/ICoCo/CMakeLists.txt
@@ -41,7 +41,9 @@ target_link_libraries(medicoco medcouplingcpp)
 install(
   TARGETS medicoco
   EXPORT ${PROJECT_NAME}TargetGroup
-  DESTINATION ${MEDCOUPLING_INSTALL_LIBS})
+  ARCHIVE DESTINATION ${MEDCOUPLING_INSTALL_LIBS}
+  RUNTIME DESTINATION ${MEDCOUPLING_INSTALL_BINS}
+  LIBRARY DESTINATION ${MEDCOUPLING_INSTALL_LIBS})
 
 file(GLOB icoco_HEADERS_HXX "${CMAKE_CURRENT_SOURCE_DIR}/*.hxx"
      "${CMAKE_CURRENT_SOURCE_DIR}/*.h")
diff --git a/src/INTERP_KERNEL/CMakeLists.txt b/src/INTERP_KERNEL/CMakeLists.txt
index f69ffcc..9152872 100644
--- a/src/INTERP_KERNEL/CMakeLists.txt
+++ b/src/INTERP_KERNEL/CMakeLists.txt
@@ -91,7 +91,9 @@ target_link_libraries(interpkernel ${PLATFORM_LIBS})
 install(
   TARGETS interpkernel
   EXPORT ${PROJECT_NAME}TargetGroup
-  DESTINATION ${MEDCOUPLING_INSTALL_LIBS})
+  ARCHIVE DESTINATION ${MEDCOUPLING_INSTALL_LIBS}
+  RUNTIME DESTINATION ${MEDCOUPLING_INSTALL_BINS}
+  LIBRARY DESTINATION ${MEDCOUPLING_INSTALL_LIBS})
 
 file(GLOB_RECURSE interpkernel_HEADERS_HXX "${CMAKE_CURRENT_SOURCE_DIR}/*.hxx")
 file(GLOB_RECURSE interpkernel_HEADERS_TXX "${CMAKE_CURRENT_SOURCE_DIR}/*.txx")
diff --git a/src/MEDCoupling/CMakeLists.txt b/src/MEDCoupling/CMakeLists.txt
index 26bebbe..4931242 100644
--- a/src/MEDCoupling/CMakeLists.txt
+++ b/src/MEDCoupling/CMakeLists.txt
@@ -93,14 +93,18 @@ target_link_libraries(medcouplingcpp interpkernel)
 install(
   TARGETS medcouplingcpp
   EXPORT ${PROJECT_NAME}TargetGroup
-  DESTINATION ${MEDCOUPLING_INSTALL_LIBS})
+  ARCHIVE DESTINATION ${MEDCOUPLING_INSTALL_LIBS}
+  RUNTIME DESTINATION ${MEDCOUPLING_INSTALL_BINS}
+  LIBRARY DESTINATION ${MEDCOUPLING_INSTALL_LIBS})
 
 add_library(medcouplingremapper ${medcouplingremapper_SOURCES})
 target_link_libraries(medcouplingremapper medcouplingcpp)
 install(
   TARGETS medcouplingremapper
   EXPORT ${PROJECT_NAME}TargetGroup
-  DESTINATION ${MEDCOUPLING_INSTALL_LIBS})
+  ARCHIVE DESTINATION ${MEDCOUPLING_INSTALL_LIBS}
+  RUNTIME DESTINATION ${MEDCOUPLING_INSTALL_BINS}
+  LIBRARY DESTINATION ${MEDCOUPLING_INSTALL_LIBS})
 
 file(GLOB medcoupling_HEADERS_HXX "${CMAKE_CURRENT_SOURCE_DIR}/*.hxx")
 file(GLOB medcoupling_HEADERS_TXX "${CMAKE_CURRENT_SOURCE_DIR}/*.txx")
diff --git a/src/MEDLoader/CMakeLists.txt b/src/MEDLoader/CMakeLists.txt
index 7ae94a0..e619d8f 100644
--- a/src/MEDLoader/CMakeLists.txt
+++ b/src/MEDLoader/CMakeLists.txt
@@ -98,7 +98,9 @@ target_link_libraries(medloader medcouplingcpp ${MEDFILE_C_LIBRARIES}
 install(
   TARGETS medloader
   EXPORT ${PROJECT_NAME}TargetGroup
-  DESTINATION ${MEDCOUPLING_INSTALL_LIBS})
+  ARCHIVE DESTINATION ${MEDCOUPLING_INSTALL_LIBS}
+  RUNTIME DESTINATION ${MEDCOUPLING_INSTALL_BINS}
+  LIBRARY DESTINATION ${MEDCOUPLING_INSTALL_LIBS})
 
 file(GLOB medloader_HEADERS_HXX "${CMAKE_CURRENT_SOURCE_DIR}/*.hxx")
 file(GLOB medloader_HEADERS_TXX "${CMAKE_CURRENT_SOURCE_DIR}/*.txx")
