diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8c5139d..3fce7fd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,12 +7,13 @@ PROJECT(MEDFile C CXX)
 #ENABLE_LANGUAGE(Fortran OPTIONAL)
 
 INCLUDE(CheckLanguage)
-CHECK_LANGUAGE(Fortran)
+#CHECK_LANGUAGE(Fortran)
 IF(CMAKE_Fortran_COMPILER)
-  ENABLE_LANGUAGE(Fortran)
+  #ENABLE_LANGUAGE(Fortran)
 ELSE()
   MESSAGE(STATUS "No Fortran support")
 ENDIF()
+set(CMAKE_Fortran_COMPILER_WORKS 0)
 
 ## Version number 
 SET(MED_NUM_MAJEUR  6)
diff --git a/config/cmake_files/medMacros.cmake b/config/cmake_files/medMacros.cmake
index ed4582f..1f9ef05 100644
--- a/config/cmake_files/medMacros.cmake
+++ b/config/cmake_files/medMacros.cmake
@@ -212,7 +212,7 @@ IF(_med_medint_type_is_set AND (NOT CMAKE_Fortran_COMPILER_SUPPORTS_F90) )
   SET(LONG_OR_INT ${MED_MEDINT_TYPE})
 ENDIF()
 
-IF(_med_medint_type_is_set AND CMAKE_Fortran_COMPILER_SUPPORTS_F90 )
+IF(_med_medint_type_is_set AND CMAKE_Fortran_COMPILER_SUPPORTS_F90 AND FALSE)
   CHECK_FORTRAN_INTEGER_SIZE()
   IF( NOT ${SIZEOF_MEDINT_TYPE} EQUAL ${SIZEOF_FORTRAN_INTEGER} )
     MESSAGE(FATAL_ERROR "The MED_MEDINT_TYPE definition must be coherent with the default fortran integer. \
